src/integrationTest/resources/sql/insert_access_types.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM access_types;
```

Comment:
just a clarification question, why do we empty the table?

Reply:
deleting the `access_types` table allows us to reuse the `@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_access_types.sql"})` for other tests

if we didn't clear the table & use this annotation for another test, the insert statement would fail - due to the data existing with same primary key